mautic/mautic
# Process email queue lock

Please answer the following questions. 

| Q | A |
| --- | --- |
| Bug fix? | Y |
| New feature? | N |
| Related user documentation PR URL | N |
| Related developer documentation PR URL |  |
| Issues addressed (#s or URLs) | #1737 #2215 |
| BC breaks? | N |
| Deprecations? | N |

**Note that all new features should have a related user and/or developer documentation PR in their respective repositories.** 
### Required
#### Description:

Avoid process **mautic:emails:send** to be run twice.
--force option can bypass this new check
#### Steps to test this PR:
1. Run php app/console mautic:emails:send & twice at the same time from console
2. The second process should return "Process already running."
### As applicable
#### Steps to reproduce the bug:

see #2215
